docs/omero_figure_move.rst

```diff
@@ -0,0 +1,99 @@
+
+#.  Repeat the step above for all the Figures containing the images you want to move. Retain the Figure IDs (for example saving them as a text document) for later use.
+
+#.  Go back to the tab with OMERO.web, create a new Dataset and select it.
+
+#.  Click on the ``Scripts`` icon |image1| above the central pane of OMERO.web. Select ``Figure scripts > Figure Images to Dataset``. Start the script and paste into the dialog which appears your Figure IDs, separated with commas and no spaces, e.g. ``79828,79830,71228``. Click ``Run``.
+
+#.  When the script finishes, refresh the page and find the images contained in your Figure linked to the Dataset. Note that the images are only doubly-linked to two Datasets now, and no independent copy of the images was done in this step.
+
+#.  The above means that when you execute the Move into another group, the images in your Dataset will no longer be available in the original group.
+
+#.  Move the images into the group you have moved the Figure to. This will enable any user permitted to see your images in the target group to view your Figure.
+
+*Duplication of images, move and re-create Figure(s) in the new group using a script*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In case you want to have both Figures and the images contained in them in both groups, the original and the target group, you can use following workflow. Note that this workflow involves usage of Command Line.
+
+#.  Retrieve the images contained in the Figures using the workflow above. For this, you can find out the Figure IDs either manually as described in the chapter :ref:`Retrieval<Retrieval>` or by running hql queries on the command line, such as::
+
+    $ omero hql --all --limit 1000 --style plain --ids-only  "select f.id from FileAnnotation f where (f.details.group.name = 'Lab1' and f.details.owner.id = 454)" | sed -e 's/^.*,//g' | paste -s -d, -
+
+    which will retrieve all the Figure IDs of user with ID 454 in a group ``Lab1`` in a format which you can immediately copy and paste into the ``Figure Images to Dataset`` script.
+
+#.  Start your command line terminal and duplicate the Dataset with the images contained in the Figures as described in `the Duplicate workflow <https://omero-guides.readthedocs.io/en/latest/introduction/docs/data-management.html#command-line-duplicating-objects>`__.
+
+#.  Go to OMERO.web, select the duplicate Dataset and Move it to the target group. For that, follow `the Move workflow <https://omero-guides.readthedocs.io/en/latest/introduction/docs/data-management.html#move-data-between-groups>`__.
+
+#.  Find the Dataset which you have just moved and select it.
+
+#.  Click on the ``Scripts`` icon |image1| above the central pane of OMERO.web. Select ``Figure scripts > Dataset Images To New Figure``. 
+
+#.  Into the dialog which appears paste your Figure IDs, separated with commas and no spaces, e.g. ``79828,79830,71228``. Click ``Run``.
```

Comment:
as above, "no spaces" is not essential. After ``Run``, you could add "This will copy each specified Figure, update the Images with it to those in the duplicate Dataset (using the Image name to match the replacement Images) and save the Figure to the new Group"

Reply:
fixed in https://github.com/ome/omero-guide-figure/pull/13/commits/4eb8f01b2bd0f64dcb0ae49158bc6592b117988c